service/src/main/java/org/databiosphere/workspacedataservice/InstanceInitializerBean.java

```
@@ -255,18 +267,19 @@ private void restoreFromRemoteBackup(String backupFileName, UUID cloneJobId) {
     private void initializeDefaultInstance() {
         try {
             UUID instanceId = UUID.fromString(workspaceId);
-
             if (!instanceDao.instanceSchemaExists(instanceId)) {
                 instanceDao.createSchema(instanceId);
                 LOGGER.info("Creating default schema id succeeded for workspaceId {}.", workspaceId);
             } else {
-                LOGGER.debug("Default schema for workspaceId {} already exists; skipping creation.", workspaceId);
+                LOGGER.debug("Default schema for workspaceId {} already exists or is in progress; skipping creation.", workspaceId);
             }
 
         } catch (IllegalArgumentException e) {
             LOGGER.warn("Workspace id could not be parsed, a default schema won't be created. Provided id: {}.", workspaceId);
         } catch (DataAccessException e) {
             LOGGER.error("Failed to create default schema id for workspaceId {}.", workspaceId);
+        } catch (Exception e) {
+            LOGGER.error("Failed to create default schema id for workspaceId {}. {}", workspaceId, e.getMessage());
```

I typically advocate to [fail fast](https://enterprisecraftsmanship.com/posts/fail-fast-principle/#:~:text=The%20fail%20fast%20principle%20stands,appear%20counter%2Dintuitive%20at%20first.) on  unhandled exceptions.  Is there something that was happening in particular that made you want to catch this class of exception rather than let it bubble up?  If possible, prefer catching and proceeding _only_ with the narrowest type possible.  If there isn't a particular exception type to work with, you can look for a regex in the exception message and rethrow anything that doesn't match.